src/main/java/software/amazon/cloudformation/AbstractWrapper.java

```diff
@@ -372,7 +372,9 @@ public void processRequest(final InputStream inputStream, final OutputStream out
 
     }
 
-    protected void writeResponse(final OutputStream outputStream, final ProgressEvent<ResourceT, CallbackT> response)
+    protected void writeResponse(final OutputStream outputStream,
```

Comment:
This seems a break change for the generated code.   I would suggest you do following to keep backward compatible.  And also update the code generate to use new method. In such way,  it won't break old code if still use old plugin, but it just won't have additional logging data. 

```
protected void writeResponse(final OutputStream outputStream,final ProgressEvent<ResourceT, CallbackT> response) {
 this(outputStream, response, null);
}